http/vibe/http/client.d

```diff
@@ -363,6 +380,7 @@ final class HTTPClient {
 	*/
 	void connect(string server, ushort port = 80, bool use_tls = false, const(HTTPClientSettings) settings = defaultSettings)
 	{
+		//FIXME: actually this function is not connects, but destroys existing connection
```

Comment:
This should be removed. Maybe the method name should eventually be changed to `setConnectionTarget` or something along those lines, but the implementation is working as intended (and it has nothing to do with the other changes). BTW, although it doesn't physically open a TCP connection, "connect" is in line with the BSD socket API when using datagram based sockets, which are in a way similar to a request based protocol like HTTP.

Reply:
Ok, sure

 It just cost me a bit of time and I decided to pay attention to this fact. I’ll delete it now.